plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/cross-sells.tsx

```diff
@@ -21,7 +21,7 @@ const collection = {
 	),
 	keywords: [ 'boost', 'promotion' ],
 	scope: [],
```

Comment:
I'm fixing the Cross-Sells scope in https://github.com/woocommerce/woocommerce/pull/52209 but In the current shape of this PR, Cross-Sells is unavailable and cannot be tested correctly so I suggest you add this (quality team is testing PRs weekly so there's a chance this PR will be tested without mine and would be perceived as broken):

```suggestion
	scope: [ 'block' ],
```

Reply:
Good point. I thought about adding it but then I thought your PR might be merged first. But it makes sense to add in this PR as well, in case your PR isn't merged first. Also, It would be easier for reviewer to review this PR.